### ISSUE TITLE
Latest version of MuseScore 4 crashes when attempting The VST Aria

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#24982]
### Issue type

Crash or freeze

### Description with steps to reproduce

https://musescore.org/en/node/369256

> 
> After updating to Version 4.4.2, attempting to play the Aria VST will crash MuseScore where it worked before the update.
> 
> When I change sound back to MSBasic it works well. The When I change back to Aria, it still works. If I then save and close MuseScore and reopen it, it will crash when truing to play the Aria sounds.

### Supporting files, videos and screenshots

1

### What is the latest version of MuseScore Studio where this issue is present?

4.4.2

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 10

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by benwiggy** [comment]
Still seeing this (or similar) in 4.6.5 on MacOS Sequoia. MuseScore app unexpectedly quits when closing the ARIA VSIT window. Doesn't happen in other host apps. No crashlogs in the OS that I can find.

**Comment by DmitryArefiev** [comment]
> Still seeing this (or similar) in 4.6.5 on MacOS Sequoia. MuseScore app unexpectedly quits when closing the ARIA VSIT window. Doesn't happen in other host apps. No crashlogs in the OS that I can find.

Yes, I **can reproduce** this crash in 4.6.5 (and 4.7) on Mac13.7.8 (doesn't occur in MS4.5.2):

https://github.com/user-attachments/assets/5c8fc010-a9e8-44eb-a2c6-ca99b0d01031


Logged separately - #32742

**Comment by CubikingChill** [comment]
@DmitryArefiev Please have a look at my fix at https://github.com/musescore/MuseScore/issues/16144 . From some user's comment it seems that my fix may resolve this issue. I have no idea how to use ARIA player. But I have no issue loading it, opening the VST UI dialog and play the score (in silence). In default state ARIA doesn't give any sound.

**Comment by CubikingChill** [comment]
May I know if this issue has been fixed?

**Comment by benwiggy** [comment]
It seems to work for me in the latest release, 4.7.5.
